doc/api/fs.md

```diff
@@ -2004,6 +2004,9 @@ changes:
 Asynchronous rmdir(2). No arguments other than a possible exception are given
 to the completion callback.
 
+*Note*: Using `fs.rmdir()` on a file (not directory) results in an `ENOENT`
+error on Windows and `ENOTDIR` error on Linux/macOS.
+
```

Comment:
thanks, also it is a good idea to generalize `ENOTDIR` for UNIX systems, not just Linux.